aya/src/programs/mod.rs

```diff
@@ -604,7 +597,7 @@ fn load_program<T: Link>(
         kernel_version: target_kernel_version,
         expected_attach_type: *expected_attach_type,
         prog_btf_fd: btf_fd.as_ref().map(|f| f.as_fd()),
-        attach_btf_obj_fd: *attach_btf_obj_fd,
+        attach_btf_obj_fd: attach_btf_obj_fd.as_ref().map(|fd| fd.as_raw_fd() as _),
```

Comment:
why not make `attach_btf_obj_fd` be a `Option<BorrowedFd<'a>>`? We already do that for `prog_btf_fd`,

Reply:
Good call. Done.